src/NuGetForUnity/Editor/NugetConfigFile.cs

```diff
@@ -74,6 +74,20 @@ public class NugetConfigFile
         /// </summary>
         public bool ReadOnlyPackageFiles { get; set; }
 
+        public string PackagesConfigPath { get; set; }
+
+        public string RelativePackagesConfigPath
+        {
+            get => NugetHelper.GetAssetsRelativePath(PackagesConfigPath);
```

Comment:
I think we should use `PathHelper.GetRelativePath` directly. This makes it clearer what happens. So the setter and the getter do the "same thing"